tools/lint.js

```diff
@@ -213,7 +213,7 @@ async function ensureNoNewITests() {
     "pm_tests.rs": 0,
     "publish_tests.rs": 28,
     "repl_tests.rs": 0,
-    "run_tests.rs": 381,
+    "run_tests.rs": 382,
```

Comment:
In the future, we should be making this number go down and not up.

Reply:
I tried going the specs way but all other jsx specs used comments to set their settings. Since this has no equivalent comment (yet?) I opted to just go with the existing itest macro